buf/registry/module/v1beta1/commit_service.proto

```diff
@@ -17,341 +17,135 @@ syntax = "proto3";
-    ModuleRef module_ref = 1 [(buf.validate.field).required = true];
-    // The FileNodes for the Module.
-    //
-    // This consists of the .proto files, license files, and documentation files. This does
-    // not contain any files relating to dependencies or configuration (such as buf.lock or
-    // buf.yaml). A server will respond with an error if an unexpected file is passed here.
-    //
-    // Blobs are shared across Modules within a given CreateCommit call; missing blobs
-    // should be passed via the missing_blobs field. See GetMissingBlobDigests for more details.
-    repeated buf.registry.storage.v1beta1.FileNode file_nodes = 2 [(buf.validate.field).repeated.min_items = 1];
-    // The dependencies of the Module.
-    repeated DepNode dep_nodes = 3;
-  }
-
-  message Value {
-    // The pointers to the content for the Modules that should have Commits created for them.
-    //
-    // Each ModuleNode must have a unique ModuleRef.
-    // A commit will be created for each ModuleNode.
-    repeated ModuleNode module_nodes = 1 [(buf.validate.field).repeated.min_items = 1];
-    // Associated VCS commit information.
-    //
-    // If there are already VCSCommits on the associated Module with a given hash, this
-    // will result in an error. Otherwise, a new VCSCommit is created.
-    repeated AssociatedVCSCommit associated_vcs_commits = 2;
-  }
-  // The requests to create Commits.
-  repeated Value values = 1 [(buf.validate.field).repeated.min_items = 1];
-  // Blobs for the FileNodes referenced by file_nodes that are not present on the server.
+message ResolveCommitsRequest {
+  // References to request a Commit for.
   //
-  // Only Blobs that were returned as missing from GetMissingBlobDigests need to be sent.
-  // Other Blobs already exist on the server, and will be ignored.
+  // See the documentation on ResourceRef for resource resolution details.
   //
-  // If a FileNode has a Blob that is not on the server, and is not
-  // within missing_blobs, an error will be returned.
-  repeated buf.registry.storage.v1beta1.Blob missing_blobs = 2;
-}
-
-message CreateCommitsResponse {
-  // The created Commits, in the order of the nodes given via module_nodes.
+  // Once the resource is resolved, this reference is interpeted to refer to the following Commit:
+  //   - If a Module is referenced, this is interpreted to mean the latest released Commit on the Module.
+  //   - If a Commit is referenced, this just references that specific commit.
+  //   - If a Tag is referenced, this is interpreted to mean the Commit associated with the Tag.
+  //   - If a VCSCommit is referenced, this is interpreted to mean the Commit associated with the VCSCommit.
+  //   - Is a Branch is referenced, this is interpreted to mean the head Commit on the Branch.
+  //   - If a Digest is referenced, this is interpreted to mean the latest released Commit that has this Digest.
+  //     Digests referencing unreleased Commits cannot be referenced.
```

Comment:
The current proposal is not to have "released" be a property of a commit. Here is what I think the docs should be:

```suggestion
  // Once the resource is resolved, this reference is interpreted to refer to the following Commit:
  //   - If a Module is referenced, this is interpreted to mean the latest Commit on the default Label of the Module.
  //     If the default Label for this module has governance flow enabled, the latest Commit that has been approved will
  //     be returned.
  //   - If a Commit is referenced, this just references that specific commit.
  //   - If a Tag is referenced, this is interpreted to mean the Commit associated with the Tag.
  //   - If a VCSCommit is referenced, this is interpreted to mean the Commit associated with the VCSCommit.
  //   - Is a Branch is referenced, this is interpreted to mean the head Commit on the Branch.
  //   - If a Digest is referenced, this is interpreted to mean the latest released Commit that has this Digest.
```

Reply:
This was mean to be deleted - deleting